### PR TITLE
5372 multiple api server replicas

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
Add 4 replicas of the api-server to all deployments.
## Related issue/s
- resolves https://github.com/ITISFoundation/osparc-simcore/issues/5372
## Checklist

- [ ] I tested and it works

### CODE REVIEW

**Comment by mrnicegyu11** [comment]
@YuryHrytsuk Does this impact some deployments where resources are tight potentially?

Thanks a lot @bisgaard-itis.

If you guys find it helpful I can contribute a further change that the number of replicas can be set in the repo.config for all deployments, if desired. Then we dont need to have replicas everywhere

**Comment by bisgaard-itis** [comment]
> @YuryHrytsuk Does this impact some deployments where resources are tight potentially?
> 
> Thanks a lot @bisgaard-itis.
> 
> If you guys find it helpful I can contribute a further change that the number of replicas can be set in the repo.config for all deployments, if desired. Then we dont need to have replicas everywhere

What do you think @mguidon? Does it make sense to go with 4 api-server replicas everywhere or should we only give it a try on sim4life.io initially?

**Comment by YuryHrytsuk** [comment]
@bisgaard-itis I would propose to go with 2 and test it on aws master

**Comment by bisgaard-itis** [comment]
> @bisgaard-itis I would propose to go with 2 and test it on aws master

Yep, makes sense, thanks. So that would mean it would have to be set in repo.config, right?

**Review by pcrespov — APPROVED** [review]
:muscle:

File: services/simcore/docker-compose.yml [hunk 1]
```diff
@@ -83,6 +83,7 @@ services:
       - public
       - monitored
     deploy:
+      replicas: 4
```
Comment (YuryHrytsuk) [inline services/simcore/docker-compose.yml:86]:
```suggestion
      replicas: 2
```

**Comment by YuryHrytsuk** [comment]
> > @bisgaard-itis I would propose to go with 2 and test it on aws master
> 
> Yep, makes sense, thanks. So that would mean it would have to be set in repo.config, right?

No. You do it correctly in this PR. You just need to change number from `4` to `2`

**Comment by bisgaard-itis** [comment]
@YuryHrytsuk let me know if I can merge for you

**Review by YuryHrytsuk — APPROVED** [review]

**Comment by YuryHrytsuk** [comment]
> @YuryHrytsuk let me know if I can merge for you

go ahead

**Pull request merged**